src/components/About.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
Generally, the layout seems simple and nice. Using linters like prettier the code could be formatted better.